bettermdptools/utils/plots.py

```diff
@@ -25,6 +25,14 @@ def v_iters_plot(data, title, show=True):
         if show:
             plt.show()
 
+    @staticmethod
+    def get_values_agg_axis_means(pi, val_max, map_size, agg_axes):
+        """Aggregate by taking means over axes of multi-dimension maps. Used to pre-process for visuals."""
+        val_max, policy_map = Plots.get_policy_map(pi, val_max, None, map_size)
```

Comment:
Could you also add descriptions of the parameters and return values in the doc string?